README.md

```diff
@@ -5,33 +5,82 @@
   <img src="./.github/assets/topos_logo_dark.png#gh-dark-mode-only" alt="Logo" width="200">
   <br />
   <p align="center">
-  <b>Topos Playground</b> is a CLI make it simple to run a local Topos devnet 🚀
+  <b>Topos Playground</b> is a CLI to make it simple to run a local Topos devnet 🚀
   </p>
   <br />
 </div>
 
 ## Getting Started
 
+The Topos Playground is a CLI that allows you to easily run and manage a local Topos devnet. It depends on Docker and NodeJS, and manages a number of components in order to locally run multiple subnets, multiple TCE nodes, and a dApp with supporting infrastructure, capable of cross-subnet messaging.
+
 ### Requirements
 
+- [Docker](https://docs.docker.com/get-docker/_) version 17.06.0 or greater
+- [NodeJS](https://nodejs.dev/en/) version 16.0.0 or greater
 - Git
-- NodeJS (tested with 16+)
-- Docker
 
-### [Optional] Install the package globally
+### Install / Run topos-playground
+
+Depending on your NodeJS environment and preferences, there are several ways to install the Topos Playground.
+
+To install the playground globally, using `npm`, run the following command:
 
 ```
 $ npm install -g @topos-protocol/topos-playground
 ```
 
+To install the playground globally, using `yarn`, run the following command:
+
+```
+$ yarn global add @topos-protocol/topos-playground
+```
+
+Alternatively, you can install and run via `npx`:
+
+```
+$ npx @topos-protocol/topos-playground
+```
+
+#### Install via Git
+
+If you prefer to install the playground via Git, you can clone the repository and install it manually.
+
+```
+$ git clone https://github.com/topos-protocol/topos-playground.git
+$ cd topos-playground
+$ npm install
+$ npm run build && npm link
+```
+
 ### Run the CLI
 
-If you have installed the package manually, you can run it like so:
+If you have installed the via `npm` or `yarn`, you can run the CLI like any other command line tool:
```

Comment:
Missing instructions on how to run the tool if you installed via git

Reply:
Added another clause:

`If you have installed the via `npm`, `yarn`, or manually after building it from the git repository...`